### ISSUE TITLE
selectValue method isn't safe

### ISSUE THREAD

**Issue by SmileHub** [kevalpatel2106/android-ruler-picker#7]
so i've discovered that selectValue method isn't safe. If you call it multiple times in a row it will do some asynchroniuos way of setting value for each all which will eventually mess up the visible value (even if you're setting up the same value all the time).

Case of usage:
1) onCreateView you setlectValue and rotate the screen.
2) your onCreateView selectValue gets called and default selectValue from restoring state gets called
3) in result if you wanted to set value 1 in range [0, 10]  your final value will be 2, if it was 2 then final value will be 4

**Comment by SmileHub** [comment]
my temporar work around:
```
public class myFragment extends DialogFragment {
    private boolean initializeValues = true;
    
    @Override
    public void onConfigurationChanged(Configuration newConfig) {
        super.onConfigurationChanged(newConfig);
        initializeValues = false;
    }

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
          View view = inflater.inflate(R.layout.fragmentX, container, false);
          RulerValuePicker P = view.findViewById(R.id.ruler_picker);
          //do some picker preparation
          if (initializeValues) {
               P.selectValue(<whatever value we wish>);
           }else
               initializeValues = true;
      return view;
    }
```
and dont forget to add to manifest something like:
`android:configChanges="orientation|screenSize"`

**Comment by kevalpatel2106** [comment]
@SmileHub I found the issue. You can check the solution in bb072520e49a402ad7d981fbf456544299e6af14.

**Comment by kevalpatel2106** [comment]
Bug is fixed in 1.1.
